### PR TITLE
feat: theming

### PR DESCRIPTION
**Pull request by xxori**

### CODE REVIEW

File: src/components/context-menus/ThemeListContextMenu.svelte [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<script lang="ts">
```
Comment (xcodian) [inline src/components/context-menus/ThemeListContextMenu.svelte]: Is this file being used anywhere?

Comment (xxori) [reply src/components/context-menus/ThemeListContextMenu.svelte]: It was leftover from before. Fixed in acab commit

**Comment by xxori** [comment]
👍

**Review by xcodian — APPROVED** [review]
LGTM, think this is ready to merge now. Please test out changes on your end and see if they work well.

0c9a612 has a few changes so take care to review them.

**Pull request merged**